graftm/pplacer.py

```diff
@@ -160,15 +208,15 @@ def check_reads(read_lists):
             raise Exception
         # Return the hash
 
-        return hash 
+        return crossover_hits 
 
-    def compare_placements(self, forward_gup, reverse_gup, hash, placement_cutoff):
+    def compare_placements(self, forward_gup, reverse_gup, crossover, placement_cutoff):
         ## Take guppy placement file for the forward and reverse reads, compare
         ## the placement, and make a call as to which is to be trusted. Return
         ## a list of trusted reads for use by the summary step in GraftM
         # Report and record trusted placements
         comparison_hash = {'trusted_placements': {}} # Set up a hash that will record info on each placement
-        for read in hash['crossover']:
+        for read in crossover:
             if read.startswith('FCC'):
                 f_read = read + '/1'
```

Comment:
I know it isn't from this merge, but this seems dangerous, I'm not sure all /1 style names start with FCC. Maybe a better way to do this would be to record in the reads are of the '/1' type when you have the full read name, and then refer to that info here.